### ISSUE TITLE
[BUG] expand_bits_to_bytes could be improved by returning ndarray of np.bool

### ISSUE THREAD

**Issue by trevorsm7** [rapidsai/cudf#4206]
**Describe the bug**
`expand_bits_to_bytes`, defined below, is a utility function for unit tests which unpacks a bitmask into a list of valids per element. As currently defined, this returns a `list` of `int` where 1 is valid and 0 is invalid. However, in every case where this is used, the caller converts the result to `ndarray` of `bool`. The input to the function is expected to be an `ndarray` anyway, so there could be some numpy transform which gives us the result as `ndarray` directly.
https://github.com/rapidsai/cudf/blob/a831f11bdeb2cb8dfcc2f47ef69b596a2334c3d2/python/cudf/cudf/tests/utils.py#L36-L43

**Expected behavior**
Refactor `expand_bits_to_bytes` to return `ndarray` of `bool`. [unpackbits](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.unpackbits.html) looks like it would be useful here.

**Comment by jrhemstad** [comment]
I think this is an old utility function that is slated to be eliminated and using https://github.com/rapidsai/cudf/issues/4142 instead. @kkraus14 correct me if I'm wrong.

**Comment by kkraus14** [comment]
I wasn't even aware of the existence of this function. It should probably use https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/utils/cudautils.py#L250 instead of rolling it's own.

**Comment by kkraus14** [comment]
The function I linked above would be replaced with #4142 as Jake mentioned 😄

**Comment by kkraus14** [comment]
Oh wait, this is a testing utility function that likely works on host objects. We probably would want to use: https://docs.scipy.org/doc/numpy/reference/generated/numpy.unpackbits.html

**Comment by trevorsm7** [comment]
Yeah, this is just an internal unit testing utility. Not really high priority. I just noticed it was odd while I was working on the hash.pyx tests.

**Comment by trevorsm7** [comment]
I looked up `unpackbits` while I was writing the description, and I agree that should work.

**Comment by mroeschke** [comment]
This function no longer exists in cuDF classic so closing
